src/main/resources/assets/presencefootsteps/lang/de_de.json

```diff
@@ -0,0 +1,67 @@
+{
+	"key.presencefootsteps.settings": "Presence Footsteps",
+	"pf.default_sounds.name": "Standard Sound Packet",
+	"menu.pf.title": "Presence Footsteps Optionen",
+	"menu.pf.multiplayer.true": "Einzel + Mehrspieler",
+	"menu.pf.multiplayer.false": "Nur Einzelspieler",
+	"menu.pf.global.all": "Entitäten: Alle",
```

Comment:
I personally think that Entitäten might not be the best way to translate Entity in this context Kreaturen or Wesen would probably work better

Reply:
Your right. I checked the Wiki and they use `Kreatur` / `Kreaturen`